README.md

```diff
@@ -71,6 +73,44 @@ This design should also be pin-compatible with the following SoCs (although this
 
 Note that not all STM32 pins are 5V-tolerant, and the PS/2 protocol is a 5V open-collector system, so ensure that whichever part you pick has 5V-tolerant pins (marked `FT` or `FTt` in the datasheet) for the PS/2 signals. All of the parts above _should_ be OK, but they haven't been tested. Let us know if you try one!
 
+### Nucleo-F401
+
+The NBMC firmware is originally designed to run on an ST Micro STM32F0 (STM32F031K6T6) microcontroller. This MCU has:
+
+* 32-bit Arm Cortex-M4 Core
+* 3.3V I/O (5V tolerant)
+* 512 KBytes Flash
```

Comment:
This README (and Neotron in general) uses KiB for Kibibytes, to avoid confusion with KB for kilobytes. So this should be KiB.